src/backend/gl/src/queue.rs

```diff
@@ -835,7 +987,26 @@ impl hal::queue::RawCommandQueue<Backend> for CommandQueue {
         S: 'a + Borrow<native::Semaphore>,
         Iw: IntoIterator<Item = &'a S>,
     {
+        let gl = &self.share.context;
+
         for swapchain in swapchains {
+            let extent = swapchain.0.borrow().extent;
+
+            gl.BlitNamedFramebuffer(
```

Comment:
This is only introduced in OGL 4.5. As a fallback, with OGL pre-4.5, bind the read and write frame buffers separately and use `glBlitFramebuffer`.

Reply:
Ah darn. I liked this function more :)